app/models/part.rb

```diff
@@ -19,7 +19,7 @@ class Part
   validates :title, presence: true
   validates :slug, presence: true
   validates :slug, exclusion: { in: %w[video], message: "Can not be video" }
-  validates :slug, format: { with: /\A[a-z0-9\-]+\Z/i }
+  validates :slug, format: { with: /\A[a-z0-9\-]+\Z/i, message: "only lower case characters, numbers and hyphens are allowed" }
```

Comment:
This still won't read correctly as a full message and should start with a verb. For example:

```
irb(main):008:0> p = Part.new(slug: "not valid"); p.valid?; p.errors.full_messages
=> ["Title can't be blank", "Slug only lower case characters, numbers and hyphens are allowed"]
```

Reply:
Ah!  Yes, I see the problem - thanks.  How about this...?

```
Slug can only consist of lower case characters, numbers and hyphens
```